snocker/app/run.sh

```diff
@@ -0,0 +1,20 @@
+# Set & move to home directory
+source set_env.sh
+cd "$SNORKELHOME"
+
+# Make sure the submodules are installed
+git submodule update --init --recursive
+
+# Make sure parser is installed
+PARSER="parser/stanford-corenlp-3.6.0.jar"
+if [ ! -f "$PARSER" ]; then
+    read -p "CoreNLP [default] parser not found- install now?   " yn
+    case $yn in
+        [Yy]* ) echo "Installing parser..."; ./install-parser.sh;;
+        [Nn]* ) ;;
+    esac
+fi
+
+# Launch jupyter notebook!
+echo "Launching Jupyter Notebook..."
+jupyter notebook --port=8887 --ip=0.0.0.0 --allow-root
```

Comment:
port 8888 is more usual?

Reply:
true, but it is all arbitrary. I was assuming that someone might have jupyter running locally. If so, 8888 could be spoken for. By choosing something less conventional, one needs simply to look at the URL and know that they are interacting with the containerized snorkel.